### DISCUSSION TITLE
Windows 10 VM (Official tests have been done!) -No longer have to do exploits.

### DISCUSSION THREAD

**Discussion by Luigi-Master** [3kh0/ext-remover#1283] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
Note:* This is just a VM launcher running on a website, not an actual VM app running. 

What good things the VM could do: 

-Run exe files

-Manually install your own extensions. 

Bad things:

-it resets after and hour and 45 minutes.

The steps:

1. copy and paste this link,  https://learn.microsoft.com/en-us/training/modules/implement-common-integration-features-finance-ops/10-exercise-1
2. Sign in with your microsoft account, if you don't have one, create one. 
3. Press launch VM, wait for 1 minute and 30 seconds for it to build. 
4. When you get to the windows 10 login screen type this password ---  pass@word1
5. Your done. You are now using windows 10.

*If your your district has fortiguard and it prevents you from accessing the blocked links in the microsoft edge browser, use other web browsers or an exe file that bypasses it. 

Your welcome

**Comment by awsome-webdev** [comment]
nice

**Comment by ghost** [comment]
what about this fortigate application control thingy that blocks stuff?
